request.go

```diff
@@ -105,7 +105,7 @@ func (r *Requester) PostXML(endpoint string, xml string, responseStruct interfac
 	if err := r.SetCrumb(ar); err != nil {
 		return nil, err
 	}
-	ar.SetHeader("Content-Type", "application/xml")
```

Comment:
ar.SetHeader("Content-Type", "application/xml;charset=utf-8")
Should solve your problem!